### ISSUE TITLE
RuboCop: Use empty lines between method definitions. is reported even if there are empy lines

### ISSUE THREAD

**Issue by agross** [brigade/haml-lint#189]
test.haml:

```haml
:ruby
  def foo
    42
  end

  def bar
    23
  end

%p
  Test
```

```sh
$ haml-lint test.haml
test.haml:6 [W] RuboCop: Use empty lines between method definitions.
```

Perhaps haml-lint should ignore this error by default?

**Comment by michaelherold** [comment]
Thanks for the report! This is because the newlines are being dropped by the Ruby extractor. I'll see if I can figure out a fix.

**Comment by agross** [comment]
👍 Many thanks!
